include/task.h

```diff
@@ -104,9 +104,11 @@ struct Task *TaskCreate(TaskMain taskMain, u16 structSize, u16 priority, u16 fla
 
 #define TaskCreate(taskMain, structSize, priority, flags, taskDestructor)               \
     ({                                                                                  \
-        printf("New '%s'(0x%X): 0x%p (in %s,%d)\n", #taskMain, (u32)structSize,         \
-               taskMain, __FILE__, __LINE__);                                           \
-        TaskCreate(taskMain, structSize, priority, flags, taskDestructor, #taskMain);   \
+        struct Task *tt = TaskCreate(taskMain, structSize, priority, flags,             \
+                                     taskDestructor, #taskMain);                        \
+        printf("New '%s' (0x%X, 0x%p) at 0x%p\n", #taskMain, (u32)structSize, taskMain, \
```

Comment:
Not gonna get a log if this crashes before we log btw, does that help you debugging?

Reply:
Fixed! And yes, it does! :D